app/views/notifications/index.haml

```diff
@@ -1,17 +1,18 @@
-.card
-  %ul.list-group#notifications
-    - if @notifications.count.zero?
-      %li.list-group-item.text-center
+%ul.list-group#notifications
```

Comment:
If there are no `list-group-items` inside, you don't need an outside `list-group` either.